lib/matplotlib/backend_bases.py

```diff
@@ -2220,7 +2220,7 @@ def get_default_filename(self):
         # Characters to be avoided in a NT path:
         # https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx#naming_conventions
         # plus ' '
-        removed_chars = r'<>:"/\|?*\0 '
+        removed_chars = r'<>:"/\|?*''\0 '
```

Comment:
I find this a bit too clever. This seems simpler
```suggestion
        removed_chars = '<>:"/\\|?*\0 '
```

Reply:
sure